### DISCUSSION TITLE
Remap Buffer Delete

### DISCUSSION THREAD

**Discussion by JonBoyleCoding** [LazyVim/LazyVim#2607] [Q&A]
I want to remap the keys for 'buffer delete' and 'buffer delete force', however I cannot seem to find where they are defined, nor which commands they are. I presume 'buffer delete' is `bdelete`, but I don't know what the force option command is.

I've also tried to delete the original keybindings, but this appears to not work.

keymaps.lua:
```lua
-- Remap Buffer Kill
vim.keymap.del("n", "<leader>bd")
vim.keymap.del("n", "<leader>bD")

vim.keymap.set("n", "<leader>bk", "<cmd>bdelete<cr>", { desc = "Kill Buffer" })
```

Any thoughts on how I can do this? (And/or do this in a better way?)

**Comment by dusty-phillips** [comment]
Looks like [mini.bufremove](https://www.lazyvim.org/plugins/editor#minibufremove) (<leader>bD is in the full spec).

**Reply by JonBoyleCoding** [reply]
Thanks!

**Reply by countcain** [reply]
wonderful!

**Reply by danielo515** [reply]
It has been removed, to don't break the tradition of introducing changes that break users configurations
